security/xca/PortFile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+PortGroup                   github 1.0
+PortGroup                   qt5 1.0
+
+github.setup                chris2511 xca 2.1.2 RELEASE.
+github.tarball_from         releases
+
+categories                  security crypto
+platforms                   darwin
+license                     Permissive
+license_noconflict          openssl
+
+maintainers                 {@lhaeger} openmaintainer
+
+description                 X Certificate and Key management
+long_description            XCA is a CA & PKI management tool that supports creating and managing X.509 certificates, \
+                            certificate requests, RSA, DSA and EC private keys, Smartcards and CRLs.
+homepage                    https://www.hohnstaedt.de/xca/
+
+checksums                   rmd160  16717ab6eb2cd33599d9606b7908551ad01bbff3 \
+                            sha256  fc845470a02b0b4534b46590be307f784662071fc412fdcad605c3bce901fe05 \
+                            size    1086563
+
+qt5.depends_component       qtdeclarative \
+                            qtmacextras \
+                            qtmultimedia \
+                            qtsvg \
+                            qttools \
+                            qttranslations \
+                            sqlite-plugin
+
+depends_lib-append          port:libtool \
+                            port:openssl
+
+patchfiles                  patch-disable-codesigning.diff
+
+configure.cppflags-append   "-I${prefix}/include -L${prefix}/lib"
```

Comment:
`-I${prefix}/include` is the default for `configure.cppflags`. See the guide section [here](https://guide.macports.org/#reference.phases.configure). So I think you only would need to add `-L${prefix}/lib`. Also, the quotes should no be necessary (or desired).